tests/neg/use-experimental-def.check

```diff
@@ -0,0 +1,10 @@
+-- Error: tests/neg/use-experimental-def.scala:7:15 --------------------------------------------------------------------
+7 |def bar: Int = foo // error
+  |               ^^^
+  |               Experimental definition may only be used under experimental mode:
+  |                 1. In a definition marked as @experimental
```

Comment:
We can make that clearer like this:
```
Experimental definition may only be used under experimental mode:
  |                 1. in a definition marked as @experimental, or
  |                 2. compiling with the -experimental compiler flag, or
  |                 3. with a nightly or snapshot version of the compiler.
```
Otherwise it could also be interpreted as "and".